backstop/tests/table.html

```diff
@@ -88,589 +91,480 @@ <h2 id="default">Default</h2>
                 </span>
               </label>
             </div>
-            <div class="iui-tables-cell iui-tables-head iui-active-sort">
+            <div
+              class="iui-cell iui-active-sort"
+              tabindex="0"
+            >
               Name
-              <div class="iui-filter">
+              <button
+                class="iui-button iui-borderless"
+                aria-label="Apply filter"
+                type="button"
+              >
                 <!-- Use filter-hollow.svg from @itwin/itwinui-icons package. -->
                 <svg
                   class="iui-icon"
-                  viewBox="0 0 16 16"
                   aria-hidden="true"
+                  viewBox="0 0 16 16"
                 >
                   <path
                     d="m15 1v.5l-5.6 4.7-.4.3v.5 5.5l-2 1.5v-7-.5l-.4-.3-5.6-4.7v-.5zm1-1h-16v2l6 5v9l4-3v-6l6-5z" />
                 </svg>
-              </div>
-              <div class="iui-sort">
-                <div class="iui-icon-wrapper">
-                  <!-- Use sort-down.svg from @itwin/itwinui-icons package. -->
-                  <svg
-                    class="iui-icon"
-                    viewBox="0 0 16 16"
-                    aria-hidden="true"
-                  >
-                    <path d="m7 0v12.7l-3.8-3.7-1.2 1.2 6 5.8 1.2-1.2 4.8-4.6-1.2-1.2-3.8 3.7v-12.7z" />
-                  </svg>
-                </div>
+              </button>
+              <!-- Use sort-down.svg from @itwin/itwinui-icons package. -->
+              <svg
+                class="iui-icon iui-sort"
+                viewBox="0 0 16 16"
+                aria-hidden="true"
+              >
+                <path d="m7 0v12.7l-3.8-3.7-1.2 1.2 6 5.8 1.2-1.2 4.8-4.6-1.2-1.2-3.8 3.7v-12.7z" />
+              </svg>
+              <div class="iui-resizer">
+                <div class="iui-resizer-bar"></div>
               </div>
```

Comment:
`.iui-resizer` is 16px wide and transparent.  When hovered, the nested `.iui-resizer-bar` grows from 2px to 4px.  `.iui-resizer` makes the touch target area a little bit easier to hover over.

Reply:
When you say "touch", I assume you really mean click?

I can't imagine anyone wanting to resize columns on a touch device

Reply:
Yeah, the area your finger or cursor could be placed to activate.  2px wide is hard to hover even with a cursor sometimes.